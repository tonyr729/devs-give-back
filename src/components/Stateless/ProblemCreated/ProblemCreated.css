.thanks-text {
  font-size: 2rem;
  padding-top: 10rem; }

.info-message {
  font-size: 4rem;
  padding: 4rem 0 8rem;
  width: 60rem; }

a p {
  font-size: 2rem;
  padding: 7rem; }
  a p:hover {
    color: #29bbff; }
