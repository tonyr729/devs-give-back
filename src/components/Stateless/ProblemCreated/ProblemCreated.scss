$grey: #4f4f4f;
$white: #ffffff;
$green: #39bda7;
$blue: #03a9f4;
$transparent: rgba(255, 255, 255, 0);
$black-05: rgba(0, 0, 0, 0.05);

.problem-created {
  background-color: #fff;
}

.thanks-text {
  font-size: 2rem;
  padding-top: 10rem;
}

.info-message {
  font-size: 4rem;
  padding: 4rem 0 8rem;
  width: 60rem;
}

a {
  p {
    font-size: 2rem;
    padding: 7rem;
    &:hover {
      color: #29bbff;
    }
  }
}

.goto-profile-button {
  background:$blue;
  color:$white;
  border:none;
  margin: 7rem auto;
  position: relative;
  border-radius: 7rem;
  display: block;
  height:60px;
  font-size:1.6em;
  padding:0 2em;
  cursor: pointer;
  transition:800ms ease all;
  outline:none;
  &:hover{
    background:$transparent;
    color:#002e42;
  }
  &:before,
  &:after {
    content:'';
    position:absolute;
    top:0;
    right:0;
    height:2px;
    width:0;
    background: $blue;
    transition:400ms ease all;
  }
  &:after {
    right:inherit;
    top:inherit;
    left:0;
    bottom:0;
  }
  &:hover:before,
  &:hover:after {
    width:100%;
    transition:800ms ease all;
  }  
}