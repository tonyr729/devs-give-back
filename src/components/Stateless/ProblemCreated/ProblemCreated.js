import React from 'react';
import { NavLink } from 'react-router-dom';
import './ProblemCreated.css';

export const ProblemCreated = () => {
  return (
    <div>
      <p className='thanks-text'>Thank you!</p>
      <p className='info-message'>Your problem has been created.</p>
      <NavLink to='/new-profile'>
        <p>Go to my profile</p>
      </NavLink>
    </div>
  );
};