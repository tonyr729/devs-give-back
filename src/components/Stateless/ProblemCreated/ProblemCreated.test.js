import { shallow } from 'enzyme';
import React from 'react';
import { ProblemCreated } from './ProblemCreated';

describe('ProblemCreated', () => {
  it('should match snapshot', () => {
    const problemCreated = shallow(<ProblemCreated  />);

    expect(problemCreated).toMatchSnapshot();
  });
});