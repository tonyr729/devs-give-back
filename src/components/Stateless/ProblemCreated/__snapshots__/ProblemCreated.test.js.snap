// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProblemCreated should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p
          className="thanks-text"
>
          Thank you!
</p>,
        <p
          className="info-message"
>
          Your problem has been created.
</p>,
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          to="/new-profile"
>
          <p>
                    Go to my profile
          </p>
</NavLink>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Thank you!",
          "className": "thanks-text",
        },
        "ref": null,
        "rendered": "Thank you!",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Your problem has been created.",
          "className": "info-message",
        },
        "ref": null,
        "rendered": "Your problem has been created.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": <p>
            Go to my profile
</p>,
          "to": "/new-profile",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Go to my profile",
          },
          "ref": null,
          "rendered": "Go to my profile",
          "type": "p",
        },
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p
            className="thanks-text"
>
            Thank you!
</p>,
          <p
            className="info-message"
>
            Your problem has been created.
</p>,
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            to="/new-profile"
>
            <p>
                        Go to my profile
            </p>
</NavLink>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Thank you!",
            "className": "thanks-text",
          },
          "ref": null,
          "rendered": "Thank you!",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Your problem has been created.",
            "className": "info-message",
          },
          "ref": null,
          "rendered": "Your problem has been created.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": <p>
              Go to my profile
</p>,
            "to": "/new-profile",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Go to my profile",
            },
            "ref": null,
            "rendered": "Go to my profile",
            "type": "p",
          },
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
